include/main.hpp

```diff
@@ -1,22 +1,33 @@
+#ifndef _MAIN_HPP_
+#define _MAIN_HPP_
+
 #include <WiFiClient.h>
 #include <WiFiServer.h>
 
 #define MAX_SRV_CLIENTS 4
 #define RXBUFFERSIZE 1024
+#define QUEUE_SIZE 480 
+#define ARBITRATION_BUFFER_SIZE 20
 #define STACK_PROTECTOR  512 // bytes
 #define HOSTNAME "esp-eBus"
 #define RESET_MS 1000
 
+
 #ifdef ESP32
+#define USE_ASYNCHRONOUS 
 // https://esp32.com/viewtopic.php?t=19788
 #define AVAILABLE_THRESHOLD 0
 #else
 #define AVAILABLE_THRESHOLD 1
 #endif
 
+inline int DEBUG_LOG(const char *format, ...) { return 0;}
+int DEBUG_LOG_IMPL(const char *format, ...);
+//#define DEBUG_LOG DEBUG_LOG_IMPL
```

Comment:
Did you notice any performance issues when logging is enabled?
Did you consider using some popular logging libs for Arduino instead of own impl?

Reply:
I did not notice any performance impact, but in production code it should probably be disabled. 
I guess better to use a standard logging api. I have doubts to include and additional dependency, also taking into account what happened with log4j

Reply:
It looks like there is some performance impact because of the logging. Difficult to notice and to be sure, it seems there is more timeouts reported by ebusd when logging is enabled